_assignments/mp1.md

```diff
@@ -176,8 +176,8 @@ The inspector tab should have a `Light` component like this:
 
 ![Light Object](/img/assignments/mp1/image6.png)
 
-Of primary importance are the range (the radius of your light), color, and intensity values.
-Set the shadow type to "soft shadows", and the mode to "Realtime".
+Of primary importance are the `Range` (the radius of your light), `Color`, and `Intensity` values.
+Set the shadow type to `Soft Shadows`, and the mode to `Realtime`.

```

Comment:
[nit] I don't like this stylistically because these are UI strings, not code objects

Reply:
The only reason I did this was because it felt inconsistent throughout the document.